pkg/webhook/mutation_test.go

```diff
@@ -0,0 +1,232 @@
+package webhook
+
+import (
+	"context"
+	"testing"
+
+	"github.com/ghodss/yaml"
+	admissionv1beta1 "k8s.io/api/admission/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	atypes "sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+)
+
+const (
+	validAssignMeta = `
```

Comment:
Should we just embed these strings in the test if we aren't going to reuse them?

Reply:
Yep. I "took inspiration" from https://github.com/open-policy-agent/gatekeeper/blob/f3ec64a56154cd991e98387329083d243c967879/pkg%2Fwebhook%2Fpolicy_test.go but using this locally will make it clearer.

Reply:
Done